producer/src/main/kotlin/streams/kafka/KafkaEventRouter.kt

```diff
@@ -95,8 +91,9 @@ class KafkaEventRouter: StreamsEventRouter {
         // in the procedures we allow to define a custom message key via the configuration property key
         // in order to have the backwards compatibility we define as default value the old key
         val key = config.getOrDefault("key", UUID.randomUUID().toString())
+        val partition = config.getOrDefault("partition", ThreadLocalRandom.current().nextInt(kafkaConfig.numPartitions)).toString().toInt()
```

Comment:
```suggestion
        val partition = config["partition"]
```

Reply:
Ho messo `(config["partition"]).toString.toInt` per non farlo andare in Cast Exception